.github/workflows/pr-correctness.yml

```diff
@@ -3,45 +3,8 @@ name: PR correctness
 on:
   pull_request:
     types: [ labeled, unlabeled, opened, synchronize, reopened ]
+  workflow_call:
 
 jobs:
   bindingtester:
-    if: contains(github.event.pull_request.labels.*.name, 'correctness')
-    name: Binding Tester
-    runs-on: ubuntu-latest
-
-    strategy:
-      matrix:
-        parallelism: ["1", "2", "3"]
-    env:
-      CARGO_INCREMENTAL: 0
-      RUST_BACKTRACE: 1
-
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@v1
-        with:
-          submodules: recursive
-
-      - name: Setup FoundationDB
-        uses: foundationdb-rs/foundationdb-actions-install@v2.0.0
-        with:
-          version: "6.3.23"
-
-      - name: Install Rust toolchain
-        uses: actions-rs/toolchain@v1
-        with:
-          toolchain: stable
-          profile: minimal
-
-      - name: Build
-        uses: actions-rs/cargo@v1
-        with:
-          command: build
-          args: -p bindingtester
-
-      - name: Setup bindingtester
-        run: scripts/setup_bindingtester.sh target/debug/bindingtester
-
-      - name: Run bindingtester
-        run: scripts/run_bindingtester.sh 1000
+    uses: foundationdb-rs/foundationdb-rs/.github/workflows/correctness.yml@main 1000
```

Comment:
I think you need to use the [`with`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepswith) syntax here:

```yml
jobs:
  bindingtester:
    name: Run bindingtester
    uses: foundationdb-rs/foundationdb-rs/.github/workflows/correctness.yml@main 1000 
    with:
      iterations: 1000
```